app/gui/src/presenter/graph/state.rs

```diff
@@ -492,6 +492,7 @@ impl<'a> ControllerChange<'a> {
             None | Some(Value) => None,
             Some(DataflowError { trace }) => Some((Kind::Dataflow, None, trace)),
             Some(Panic { message, trace }) => Some((Kind::Panic, Some(message), trace)),
+            Some(Pending { message: _, progress: _ }) => Some((Kind::Panic, Some("Pending...".to_string()), Vec::new())),
```

Comment:
The IDE needs to do something better here. Not mark the node as `Panic`, but render it _differently_ when its value is pending.

Reply:
Depending on the timeline and when this will be rolled out, would it make more sense to hide this feature until it is properly implemented in the IDE? So instead of showing a panic, maybe we should just emit a log message.

But this somewhat depends on whether this workaround is acceptable to be here until this gets the correct IDE implementation. Question for @wdanilo, probably.

Reply:
Yes, using the _"panic state"_ for a pending node is very aggressive. It'd be better to draw the node as _greyed out_ (just like it is after opening new project, before computations are provided by the engine). However I need an advice how to set such _greyed out_ state in the IDE code.

Disabling the _"panic state"_ is easy on the hand, I can do that myself without any advice.

Reply:
I agree, displaying this as panic is kind of a regression in GUI. Either this will be implemented properly in this PR (which probably should not, as we are soon workingo n improving nodes view), or this sohuld be just implemented as API that can be printed to the console for now.

Reply:
OK, thank you. Visual notification disabled in [a1165aa](https://github.com/enso-org/enso/pull/3729/commits/a1165aa67de323f46d69c38bfdd1507da551411c)